staging/cse/windows/containerdfunc.ps1

```diff
@@ -106,21 +106,8 @@ function CreateHypervisorRuntimes {
   return $hypervRuntimes
 }
 
-function GetContainerdTemplatePath {
-  Param(
-    [Parameter(Mandatory = $true)][string]
-    $ContainerdVersion
-  )
-  # sample values of version is 1.7.9 or 2.0.0
-  if (([version]$ContainerdVersion).CompareTo([version]"2.0.0") -ge 0) {
-    return "c:\AzureData\windows\containerd2template.toml"
-  }
-  return "c:\AzureData\windows\containerdtemplate.toml"
-}
 function ProcessAndWriteContainerdConfig {
   Param(
-    [Parameter(Mandatory = $true)][string]
-    $TemplatePath,
```

Comment:
Changing the method APIs in these is very risky as old CSE versions might call the new cse. Maybe. I'm not sure. But can we keep the method APIs the same anyway?